components/library/gsl/Makefile

```diff
@@ -14,35 +14,34 @@
 
 include ../../../make-rules/shared-macros.mk
 
-COMPONENT_NAME			= gsl
-COMPONENT_VERSION		= 2.1
-COMPONENT_SUMMARY		= GSL - GNU Scientific Library
-COMPONENT_SRC			= $(COMPONENT_NAME)-$(COMPONENT_VERSION)
-COMPONENT_ARCHIVE		= $(COMPONENT_SRC).tar.gz
-COMPONENT_ARCHIVE_HASH	= \
-	sha256:59ad06837397617f698975c494fe7b2b698739a59e2fcf830b776428938a0c66
-COMPONENT_ARCHIVE_URL	= ftp://ftp.gnu.org/gnu/gsl/$(COMPONENT_ARCHIVE)
-COMPONENT_FMRI			= library/math/gsl
+COMPONENT_NAME=           gsl
+COMPONENT_VERSION=        2.3
+COMPONENT_SUMMARY=        GSL - GNU Scientific Library
+COMPONENT_SRC=            $(COMPONENT_NAME)-$(COMPONENT_VERSION)
+COMPONENT_ARCHIVE=        $(COMPONENT_SRC).tar.gz
+COMPONENT_ARCHIVE_HASH=   sha256:562500b789cd599b3a4f88547a7a3280538ab2ff4939504c8b4ac4ca25feadfb
+COMPONENT_ARCHIVE_URL=    ftp://ftp.gnu.org/gnu/gsl/$(COMPONENT_ARCHIVE)
+COMPONENT_FMRI=           library/math/gsl
```

Comment:
Please, fix variables, affecting package meta-information, to match what they currently set to.